quicksort/quickSort.js

```diff
@@ -0,0 +1,39 @@
+/*
+*	Implementation of quicksort algorithm
+*	Time complexity: O(nlogn)
+*	Space complexity: O(n)
+*/
+function partition (arr, start, end) {
+	let tmp;
+	for (let i = start; i < end; i++) {
+		if (arr[i] <= arr[end]) {
```

Comment:
Please randomize the pivot. See python implementation for how  to randomize the pivot.